packages/edit-widgets/package.json

```diff
@@ -25,6 +25,7 @@
 		"@wordpress/block-editor": "file:../block-editor",
 		"@wordpress/components": "file:../components",
 		"@wordpress/element": "file:../element",
+		"@wordpress/format-library": "file:../format-library",
```

Comment:
Should we remove this?

Reply:
Yes, I think we should, I only added it because the edit-post also contained but I think we can also remove it from there. I created PR to do that https://github.com/WordPress/gutenberg/pull/16034.